.github/workflows/release.yml

```diff
@@ -130,22 +137,45 @@ jobs:
           trust_level: 5
 
       - name: Set default GPG key
-        run: |
-          gpg -K
-          gpg --list-secret-keys
-          echo "default-key ${{ steps.gpg.outputs.keyid }}" >> ~/.gnupg/gpg.conf
-          cat ~/.gnupg/gpg.conf
+        run: echo "default-key ${{ steps.gpg.outputs.keyid }}" >> ~/.gnupg/gpg.conf
 
       - name: Sign release
         run: |
           sudo apt-get install -y lowdown
           ./configure
-          # TODO: configure GPG key
           tools/build-release.sh --without-zip sign
+          mv release/SHA256SUMS.asc${{ steps.gpg.outputs.keyid  }} release/SHA256SUMS.asc
 
       - name: Upload signed artifact
         uses: actions/upload-artifact@v4
         with:
           name: c-lightning-${{ env.version }}
           overwrite: true
           path: release/
+
+      - name: Determine release data
```

Comment:
nit: this should be in a separate commit from the previous changes in this file.